arches/app/etl_modules/branch_excel_importer.py

```diff
@@ -22,14 +22,21 @@
 
 
 class BranchExcelImporter(BaseImportModule):
-    def __init__(self, request=None, loadid=None, temp_dir=None):
-        self.request = request if request else None
-        self.userid = request.user.id if request else None
+    def __init__(self, request=None, loadid=None, temp_dir=None, params=None):
+        self.loadid = request.POST.get("load_id") if request else loadid
+        self.userid = request.user.id if request else 1
```

Comment:
We can't count on there being a user 1. In fact, it's probably wise for admins to remove that user for security reasons.